platform/v1alpha1/types.go

```diff
@@ -0,0 +1,86 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type Package struct {
+	// Name contains the desired OLM-based Operator package name
```

Comment:
Nit, please check all comments, the first word should be the json tag not the Go field name. This is an OpenShift convention so that generated documentation is accurate for the end user, rather than following the GoDoc convention